src/algebra/field/ulift.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
```

Comment:
I think you could reasonably claim ownership of this file, nothing here is novel and you rewrote all of it

Reply:
Sure, why not.